### DISCUSSION TITLE
RAG on private datasets using OpenAI key and embedding model. Does OpenAI get access to the documents?

### DISCUSSION THREAD

**Discussion by iAMSagar44** [spring-projects/spring-ai#390] [Q&A]
During the entire process of Retrieval Augmented Generation (ETL, prompt stuffing and sending the data to the AI model), does OpenAI have access to the document and does it get exposed to ChatGPT?
Does OpenAI use this data to train or improve its AI models or update its data set?

**Comment by iamroberthanson** [comment]
This question isn't really related to this project.  I suggest referring to the ToS and privacy policies of whatever services you use.

Here is a link to OpenAI's current policies.
https://openai.com/policies

Note that these policies may change over time.
